mlflow/server/js/src/experiment-tracking/components/CompareRunView.js

```diff
@@ -230,10 +310,44 @@ export class CompareRunView extends Component {
     );
   }
 
+  onMouseEnterTableCell(e) {
+    var hoverTextElem = document.getElementById("table-cell-hover-text");
+    hoverTextElem.style.visibility = 'visible';
+    hoverTextElem.style.display = 'block';
+    hoverTextElem.style.left = `${e.clientX + window.scrollX}px`;
+    hoverTextElem.style.top = `${e.clientY + window.scrollY}px`;
+    hoverTextElem.innerHTML = e.target.innerHTML
+  }
```

Comment:
Looks like we tweak css a lot. Is there any ant design component that provides features we require?

Reply:
Removed this, replace it with antd Tooltip component.